client/app/assets/less/redash/ant.less

```diff
@@ -36,3 +42,35 @@
 .ant-dropdown-in-bootstrap-modal {
   z-index: 1050;
 }
+
+// Make Ant labels more Redash like
+.ant-form-item-label label {
+  color: inherit;
```

Comment:
Why is it set to white in the first place?

Reply:
No idea :confused:. In `antd` files it comes from `@label-color: @heading-color;` (but should be `fade(#000, 85%)`, not white). Anyway, I'll see what happens after update `antd`.